### ISSUE TITLE
Maestro crashes when passing an iOS udid argument

### ISSUE THREAD

**Issue by alteral** [mobile-dev-inc/maestro#509]
## Environment

Maestro: v1.17.2
Xcode: v14.1.0
MacOS: v12.5.1

## Run

```bash
maestro --udid="9E79ADD3-9558-4DCE-87EF-B9245D22434F" test flow.yaml
```

## Stacktrace

```bash
com.github.michaelbull.result.UnwrapException: kotlin.Unit io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
	at maestro.drivers.IOSDriver.open(IOSDriver.kt:57)
	at maestro.Maestro$Companion.ios(Maestro.kt:504)
	at maestro.Maestro$Companion.ios(Maestro.kt:499)
	at maestro.cli.util.MaestroFactory.createMaestro(MaestroFactory.kt:49)
	at maestro.cli.command.TestCommand.call(TestCommand.kt:78)
	at maestro.cli.command.TestCommand.call(TestCommand.kt:37)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
	at picocli.CommandLine.access$1200(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
	at picocli.CommandLine.execute(CommandLine.java:2058)
	at maestro.cli.AppKt.main(App.kt:123)
```

**Comment by reck753** [comment]
Hi all 👋 

I am experiencing the same issue. Any progress?

**Comment by artem888** [comment]
@reck753 the issue is fixed within the maestro-cli release 1.19.5
